ciphey/basemods/Decoders/braille.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional, Dict, List
+from loguru import logger
+from ciphey.iface import ParamSpec, Config, T, U, Decoder, registry
+
+
+@registry.register_multi((str, str), (bytes, bytes))
+class Braille(Decoder[T, U]):
+    UNICODE_STRING = "⠀⠁⠂⠃⠄⠅⠆⠇⠈⠉⠊⠋⠌⠍⠎⠏⠐⠑⠒⠓⠔⠕⠖⠗⠘⠙⠚⠛⠜⠝⠞⠟⠠⠡⠢⠣⠤⠥⠦⠧⠨⠩⠪⠫⠬⠭⠮⠯⠰⠱⠲⠳⠴⠵⠶⠷⠸⠹⠺⠻⠼⠽⠾⠿"
+    C_STRING = " a1b'k2l@cif/msp\"e3h9o6r^djg>ntq,*5<-u8v.%[$+x!&;:4\\0z7(_?w]#y)="
```

Comment:
This should go into cipheydists, we have documentation for this here :) https://github.com/Ciphey/Ciphey/wiki/What-to-check-before-a-pull-request#dictionaries-or-lists

Reply:
Will do.

Reply:
while moving this dictionary to cipheydists should we create a pull request?